Tests/Util/PasswordUpdaterTest.php

```diff
@@ -52,11 +54,14 @@ public function testUpdatePassword()
         $this->assertNull($user->getPlainPassword(), '->updatePassword() erases credentials');
     }
 
-    public function testUpdatePasswordWithBCrypt()
+    public function testUpdatePasswordWithEncoder()
```

Comment:
I would say `withSaltingEncoder`, not just `withEncoder` (the other case also uses an encoder)